doc/releases/release-notes-3.3.rst

```diff
@@ -474,6 +474,23 @@ Drivers and Sensors
 
 * I2C
 
+  * SAM0 Fix spurious trailing data by moving stop condition from thread into ISR
```

Comment:
To be consistent with the rest of the release notes, please use the past tense. Also, adding a colon between the platform name and the change description would improve readability.

```suggestion
  * SAM0: Fixed spurious trailing data by moving stop condition from thread into ISR
```